test-suite/Makefile

```diff
@@ -511,21 +511,21 @@ $(addsuffix .log,$(wildcard output-coqchk/*.v)): %.v.log: %.v %.out $(PREREQUISI
 
 .PHONY: approve-output
 approve-output: output output-coqtop output-coqchk
-	$(HIDE)for f in $(wildcard $(addsuffix /*.out.real,$^)); do \
+	$(HIDE)for f in $(addsuffix /*.out.real,$^); do if [ -f "$$f" ]; then \
```

Comment:
Removal of `wildcard` is fine with me. But adding `if [ -f "$$f" ]` does not make sense to me. The shell is responsible for expanding shell's wildcards. So, testing for file existence is redundant here. Same issue with all the other occurrences.

Reply:
If the glob matches nothing some shells leave it un-expanded. See #12863 #12864